src/roswire/common/package.py

```diff
@@ -160,7 +166,7 @@ def from_paths(files: dockerblade.FileSystem,
 
         Parameters
         ----------
-        files: dockerblade.FileSystem
+        app_instance: AppInstance
             access to the filesystem.
```

Comment:
Doc string above is correct, but this line is no longer accurate

Reply:
Fixed